cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -96,29 +96,52 @@ load_etc_os_release() {
   info "Running on COS build id ${BUILD_ID}"
 }
 
+reboot() {
```

Comment:
We should probably rename this, so we don't get confused with the reboot program.

Reply:
Done.